Nuxi not found

### Environment

Nuxt CLI v3.0.0-27258467.4e424d0                                                                                                                                                                                                 16:49:13
RootDir: D:\Programming\JabRefOnline                                                                                                                                                                                             16:49:14
Nuxt project info:                                                                                                                                                                                                               16:49:14

------------------------------
- Operating System: `Windows_NT`
- Node Version:     `v17.0.0`
- Nuxt Version:     `2.16.0-27257777.293dc11a`
- Package Manager:  `yarn@1.22.15`
- Bundler:          `Webpack`
- User Config:      `target`, `head`, `css`, `plugins`, `components`, `tailwindcss`, `buildModules`, `modules`, `serverMiddleware`, `watch`, `build`, `storybook`, `createRequire`
- Runtime Modules:  `-`
- Build Modules:    `@nuxtjs/tailwindcss@4.2.1`, `portal-vue/nuxt`, `@nuxt/postcss8@1.1.3`, `@pinia/nuxt@0.0.9`, `@nuxt/bridge@3.0.0-27258467.4e424d0`
------------------------------

### Describe the bug

Trying to run `yarn dev` which is defined as `nuxi dev` results in 
> 'nuxi' is not recognized as an internal or external command,

Directly running `npx nuxi` yields
> npm ERR! code EWORKSPACESCONFIG
npm ERR! workspaces config expects an Array

However, `npx nuxi-edge` works.



### Reproduction

https://github.com/JabRef/JabRefOnline/pull/480

### Additional context

It worked a couple of days ago, but stopped working now after upgrading to the latest version. I tried to delete the yarn lock and node_modules folder to no effect.

### Logs

_No response_

--- a/packages/bridge/bin/nuxt.mjs
+++ b/packages/bridge/bin/nuxt.mjs
@@ -1,0 +1,2 @@
+#!/usr/bin/env node
+import 'nuxi/cli'

--- a/packages/bridge/package.json
+++ b/packages/bridge/package.json
@@ -11,6 +11,9 @@
     "types.d.ts",
     "dist"
   ],
+  "bin": {
+    "nuxi": "./bin/nuxt.mjs"
+  },
   "scripts": {
     "prepack": "unbuild"
   },

